app/scripts/metamask-controller.actions.test.js

```diff
@@ -172,14 +172,14 @@ describe('MetaMaskController', function () {
 
       await metamaskController.createNewVaultAndKeychain('test@123');
       await Promise.all([
-        metamaskController.importAccountWithStrategy('Private Key', [
+        metamaskController.importAccountWithStrategy('privateKey', [
```

Comment:
Nit: It might be nice if we can import an enum and use that here (and other places) so we can see the significance of this value. Not sure if that is possible or not though.